R/pb_download.R

```diff
@@ -116,14 +116,16 @@ gh_download_asset <- function(owner,
     )
   }
 
+  auth_token <- if(.token != "") httr::add_headers(paste("token",.token))
```

Comment:
Yep, could steal the testthat reprex I provided earlier for sure. Just had to leave my desk but I’ll add it when I get back

Reply:
Hmm. You have a test for without authentication already https://github.com/ropensci/piggyback/blob/a51a97dcf85fcdbe8cc880ead1f9a8c54021412d/tests/testthat/test-without-auth.R#L77

but I think it's borked because you compare `datasets::iris[[2]]` to `iris[[2]]` without reading in the downloaded data to the iris variable first so that test can't actually fail

Reply:
Have amended test to read.delim from the tmpfile, that should cover this case going forward

Reply:
:+1: and good catch